test/_interface/test_amber_interface.py

```diff
@@ -1075,6 +1076,39 @@ def test_reduce_path_in_mdin():
     result = ai._reduce_path_in_mdin(test_path)
     assert len(result) <= 70
 
+def test_get_rmsd():
+    """Test the function using RMSD value calculated by CppTraj script."""
+    prmtop_path = os.path.join(MM_DATA_DIR, "test_rmsd.prmtop")
+    traj_path = os.path.join(MM_DATA_DIR, "test_rmsd.mdcrd")
+    ref_pdb = os.path.join(MM_DATA_DIR, "test_rmsd_chainid.pdb")
+
+    structure_ensemble = interface.amber.load_traj(
+        prmtop_path=prmtop_path,
+        traj_path=traj_path,
+        ref_pdb=ref_pdb,
+    )
+    region_pattern = "(br. resi 254 around 4 & polymer.protein) and (not elem H)"
+    stru_sele = select_stru(structure_ensemble.structure_0, pattern=region_pattern)
+
+    answer = [  # Calculate the rmsd values manually using cpptraj.
+        0.5866,
+        0.5253,
+        0.6249,
+        0.4517,
+        0.8457,
+        0.5998,
+        0.6588,
+        0.6615,
+        0.5346,
+        0.6015,
+    ]
+    result = interface.amber.get_rmsd(
+        stru_esm=structure_ensemble,
+        stru_selection=stru_sele,
+    )
+    for r, a in zip(result[:len(answer)], answer):
+        assert np.isclose(r, a, atol=0.001)
+
```

Comment:
clean up the temp mdcrd prmtop file generated in this test

Reply:
Fixed.